### PR TITLE
fixed typo in "heirarchy"

### PR DESCRIPTION
**Pull request by boris-kuz**

### CODE REVIEW

**Comment by p00f** [comment]
Can you rename the file too? The require will fail otherwise

**Comment by boris-kuz** [comment]
Oh right, missed that completely, sorry. Done :)

File: lua/clangd_extensions/init.lua [hunk 1]
```diff
@@ -15,7 +15,7 @@ if !exists(':ClangdAST')
   command ClangdDisableInlayHints lua require('clangd_extensions.inlay_hints').disable_inlay_hints()
   command ClangdToggleInlayHints lua require('clangd_extensions.inlay_hints').toggle_inlay_hints()
   command -range ClangdAST lua require('clangd_extensions.ast').display_ast(<line1>, <line2>)
-  command ClangdTypeHeirarchy lua require('clangd_extensions.type_heirarchy').show_heirarchy()
+  command ClangdTypeHeirarchy lua require('clangd_extensions.type_hierarchy').show_hierarchy()
```
[the remark this answers is not in the document]
Comment (boris-kuz) [reply lua/clangd_extensions/init.lua:18]: Ah, good catch. I only replaced the lower cases.

**Comment by p00f** [comment]
Thanks!

**Pull request merged**